examples/overlays/custom-overlay-default-style/index.html

```diff
@@ -0,0 +1,77 @@
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>BPMN Visualization - Custom default style for the overlays</title>
+    <link rel="shortcut icon" href="../../favicon.ico">
+    <link rel="stylesheet" href="../../static/css/spectre.min.css" type="text/css">
+    <link rel="stylesheet" href="../../static/css/icons.min.css" type="text/css">
+    <link rel="stylesheet" href="../../static/css/main.css" type="text/css">
+
+    <style>
+        .bpmn-container {
+            /* use absolute values for height to ensure that the vertical diagram is not fully displayed when the page is opened. */
+            /* width is relative to the window size */
+            height: 200px;
+        }
+    </style>
+</head>
+<body>
+    <header class="navbar bg-dark p-2">
+        <section class="navbar-section ml-2">
+            <img src="../../static/img/logo_64x64.ico" alt="logo" class="logo">
+            <span class="h3 text-light ml-2 lib-name-text">BPMN Visualization Examples</span>
+        </section>
+        <section class="navbar-section">
+            <a href="https://process-analytics.github.io/bpmn-visualization-js/" class="mr-2 btn btn-secondary">Docs</a>
+            <div class="dropdown mr-2 dropdown-right">
+                <span class="btn btn-primary dropdown-toggle" tabindex="0">
+                    GitHub <span class="icon icon-caret icon-in-the-middle"></span>
+                </span>
+                <ul class="menu">
+                    <li><a href="https://github.com/process-analytics/bpmn-visualization-js" class="mr-2 btn btn-link">Library</a></li>
+                    <li><a href="https://github.com/process-analytics/bpmn-visualization-examples" class="mr-2 btn btn-link">Examples</a></li>
+                </ul>
+            </div>
+        </section>
+    </header>
+    <section class="container col-10 flex-centered mt-2">
+            <section class="col-12 mt-2">
+
+                <div id="bpmn-container-default" class="col-12 mb-2 bpmn-container">
+                    <h2>Default style for the overlays</h2>
+                </div>
+                <div id="bpmn-container-custom-default-font" class="col-12 mt-40 mb-2 bpmn-container">
+                    <h2>Custom default font for the overlays</h2>
+                </div>
+                <div id="bpmn-container-custom-default-fill" class="col-12 mt-40 mb-2 bpmn-container">
+                    <h2>Custom default fill for the overlays</h2>
```

Comment:
```suggestion
                    <h2>Overlays custom default fill color</h2>
```

Reply:
The title that you propose is too complicated to understand what it is.

Generally, when there is more than 3 words, Mickey split the sequence.


And I didn't specify 'fill color', because we can have other field to customize for 4Fill' in the future. I don't want to refactor again later ^^

Reply:
my title proposition has less words than the existing one.
for now, this example is about the fill color, if we change what is shown in the example in the future we will change the title

Reply:
I prefer to specify it's for the overlays, because @alachambre wanted to use those properties to change the style of the Shape & the Edge.

Reply:
Done